Cargo.toml

```diff
@@ -7,3 +7,9 @@ debug = true
 
 [profile.release]
 debug = true
+
+[patch."crates-io"]
```

Comment:
I think this is fine for making sure CI works. But could you please consider not to merge this before the actual rustls release? For some people the github dependencies don't work, and it's a not a great place to have this non-buildable state for an indefinite time (until rustls actually gets published)

Reply:
I'm fine with deferring it for now.